config/initializers/dor_config.rb

```diff
@@ -17,12 +17,9 @@
     url Settings.solrizer_url
   end
 
+  # SURI url currently only used for okcomputer;  it's essential for registering objects
```

Comment:
I think we can remove the OKComputer check too. There is no direct dependency.

Reply:
will do.